src/sentry_path.h

```diff
@@ -211,13 +211,19 @@ sentry_path_t *sentry__path_from_wstr(const wchar_t *s);
  */
 sentry_path_t *sentry__path_join_wstr(
     const sentry_path_t *base, const wchar_t *other);
+
+/**
+ * This will return a copy of the last path segment, which is typically the file
+ * or directory name. The returns will utf8 string
+ */
+const char *sentry__path_filename_str(const sentry_path_t *path);
```

Comment:
this should technically be a `char *` without the const, but as commented above, I would rather avoid having this function at all.

Reply:
removed